src/lib/features_at.js

```diff
@@ -24,5 +25,14 @@ module.exports = function(event, bbox, ctx) {
       return META_TYPES.indexOf(feature.properties.meta) !== -1;
     });
 
-  return sortFeatures(features);
+  var featureIds = new StringSet();
+  var uniqueFeatures = [];
+  features.forEach((feature) => {
+    const featureId = feature.id || feature.properties.id;
```

Comment:
oh, this is in `features_at`. We should not use `feature.id`. While it should always be undefined the vector-tile-spec says it must be a number so we don't use it and thus there is no reason to check it. `feature.properties.id` is the right `id` in this case.